.travis.yml

```diff
@@ -1,10 +1,10 @@
 language: elixir
 
 elixir:
-  - 1.6.0
+  - 1.8.2
 
 otp_release:
-  - 20.2.2
+  - 22.3.4
```

Comment:
Same. 

Are they needed for this PR ???

Reply:
Same answer to the `.travis.ci.yml#elixir` update.